signinapp/templates/event.html.jinja2

```diff
@@ -48,7 +48,7 @@
       <div class="bg-secondary rounded-3 py-3 content-primary float-start">
         {# justify-content-between to get QR take up only the space it needs #}
         <div class="d-inline-flex justify-content-center m-auto hidesmall width-100" id="reader">
-          <video id="video" style="border: 1px solid gray"></video>
+          <button id="launchButton" onclick="initCamera()">Start Scanning</button>
```

Comment:
Can we have this start automatically if the user has the `can_display` role? Otherwise we'll have to click this on the kiosk all the time...

Reply:
You can only get to this page if you have the display role anyways, so that's a bit redundant. The display role really means that you can display pages.
I see it as a feature anyways - in theory most students should be scanning the code on their own phones, so we should only need to start the camera if there's a special case

Reply:
Technically the `display` role has the `autoload`  action and only the kisok user has that role.

Having to manually press the button on the Kiosk will be annoying, but point taken, pretty much everyone was scanning the code on their device anyway.